book/layout.md

```diff
@@ -55,7 +55,6 @@ class Browser:
         # ...
         self.document = Layout(self.nodes)
         self.document.layout()
-        self.display_list = self.document.display_list
```

Comment:
This is in lab5.py. Isn't it still needed?

Reply:
Is it? The corresponding code in `lab5.py` is:
```
class Browser:
    def load(self, url):
        # ...
        self.display_list = []
        paint_tree(self.document, self.display_list)
        # ...
```